.gitignore

```diff
@@ -57,6 +57,12 @@ typings/
 # dotenv environment variables file
 .env
 
-environment.json
+./environment.json
```

Comment:
Does this actually matter?  I would assume git would be smart enough to understand location without ./

Reply:
I checked in an "environment.json" to the test directory.  I'd like that to be OK, but not an "environment.json" in the root dir.

Having said that, I did this wrong anyway.  It should be "/environment.json".